common-rest/src/main/java/uk/ac/ebi/uniprot/api/rest/output/converter/AbstractUUWHttpMessageConverter.java

```diff
@@ -4,14 +4,17 @@
 import org.springframework.http.HttpInputMessage;
 import org.springframework.http.HttpOutputMessage;
 import org.springframework.http.MediaType;
+import org.springframework.http.converter.AbstractGenericHttpMessageConverter;
 import org.springframework.http.converter.AbstractHttpMessageConverter;
 import org.springframework.http.converter.HttpMessageNotReadableException;
 import org.springframework.http.converter.HttpMessageNotWritableException;
-
+import org.springframework.lang.Nullable;
```

Comment:
needed?

Reply:
I am overriding the method exactly how is used by spring with this Nullable. I think I will keep it. okay?